packages/client/src/client.ts

```diff
@@ -180,15 +186,18 @@ const createTypedApi = <D extends ChainDefinition>(
 
   return {
     query,
-    txFromCallData: (callData: Binary, token?: CompatibilityToken) =>
+    txFromCallData: (
+      callData: Binary,
+      token?: CompatibilityToken | RuntimeToken,
+    ) =>
       token
         ? _callDataTx(callData, token)
         : compatibilityToken.then((t) => _callDataTx(callData, t)),
     tx,
     event,
     apis,
     constants,
-    compatibilityToken,
+    ...(unsafe ? { runtimeToken: compatibilityToken } : { compatibilityToken }),
   } as any
```

Comment:
if `unsafe` is only being used to change this property, I think it's better for long-term maintenance to remove this flag and do this rename where this is being called. It's just one argument/flag less to keep track of.

Reply:
Sounds good.